### PR TITLE
Add new styling for the filter

### PR DESCRIPTION
**Pull request by laveesh**

### CODE REVIEW

**Comment by Ry-DS** [comment]
There's a bug: When you click the `x` on the far right instead of removing the filters one by one, the map goes blank. Think we can fix that? 

Edit: Fixed in https://github.com/Korusuke/MLH-Fellow-Map/pull/64/commits/e6e3696c45e636c95767128dce41a4964822d85e

Also think its possible to add images next to the filter options? They could be the mlh logo for now, but later on once we work it out, we could use the pod logos

File: src/components/Filter.tsx [hunk 1]
```diff
@@ -1,34 +1,32 @@
...
+  }));
 
-  function handleLayer(event: { target: { name: string; checked: boolean } }) {
-    const keyx = event.target.name;
-    const newLayers = { ...layers };
-    newLayers[keyx] = event.target.checked;
+  function handleLayer(event: any) {
+    const selectedLayers = _.map(event, (layer) => layer.label);
+    const newLayers: { [x: string]: boolean } = {};
+    _.forEach(selectOptions, (option) => {
```
Comment (SimplyBallistic) [inline src/components/Filter.tsx:14]: should we be using lodash commands here if the default js arrays support them? If we aren't using any special lodash commands, we could remove it entirely and reduce our bundle size!

**Comment by Ry-DS** [comment]
Also, selector becomes incredibly small if you resize to mobile 😥

File: src/components/Filter.tsx [hunk 2]
```diff
@@ -1,34 +1,42 @@
...
+        : true;
+    });
     setLayers(newLayers);
   }
 
   return (
-    <div id="filters">
-      <Form>{y}</Form>
+    <div className="filter">
+      <Select
```
Comment (Ry-DS) [inline src/components/Filter.tsx]: If we could change the text inside the box to say 'Filter...' that would be great!

**Review by Ry-DS — CHANGES_REQUESTED** [review]

**Comment by laveesh** [comment]
Supercede by #67

**Pull request closed without merging**